river/covariance/emp.py

```diff
@@ -183,27 +183,79 @@ def update_many(self, X: pd.DataFrame):
             )
         }
 
-        for i, j in itertools.combinations(sorted(mean.keys()), r=2):
+        self._update_from_state(n=n, mean=mean, cov=cov)
+
+    def _update_from_state(self, n: int, mean: dict, cov: float | dict):
```

Comment:
Hi @MarekWadinger. Although `ddof` is listed in the docstring, it does not appear as a parameter.

Reply:
Hello @smastelini. Thank you for point out to this. It shouldn't be in the docstring. Entry was removed in the last commit [#40598](https://github.com/online-ml/river/pull/1471/commits/40598711f1b14ead274a9bc0cd476cd247510c0f)